pkg/controller/compliancesuite/compliancesuite_controller.go

```diff
@@ -103,6 +103,22 @@ func (r *ReconcileComplianceSuite) Reconcile(request reconcile.Request) (reconci
 		return reconcile.Result{}, err
 	}
 
+	// examine DeletionTimestamp to determine if object is under deletion
+	if suite.ObjectMeta.DeletionTimestamp.IsZero() {
```

Comment:
Would be good to function-ize since it's shared with compliancescan_controller

Reply:
Although that might be overkill since it's working on a different object.. up to you

Reply:
I think I'll leave this as-is